serviceinsight/self-signed-certificate-connection.md

```diff
@@ -0,0 +1,24 @@
+---
+title: SSL Validation
+reviewed: 2016-09-28
+summary: ServiceInsight refuses to connect to a ServiceControl instance running on SSL with self-signed certificate.
+component: ServiceInsight
+tags:
+- SSL
+- ServiceControl
+- Certificate
+---
+
+ServiceControl can be configured to run on a SSL, but if a self-signed certificate is chosen, ServiceInsight would not allow connections to that instance. While running a self-sign certificate is not recommended, ServiceInsight provides an override configuration entry to allow you to connect to such instances. 
```

Comment:
> can be configured to run on a SSL

run **over** SSL

Reply:
>  While running a self-sign certificate is not recommended

Should we explain why?

Reply:
Discussed with @Particular/serviceinsight-maintainers. We will hyperlink "running a self-sign certificate is not recommended".